core/modules/image/image.field.inc

```diff
@@ -162,18 +163,31 @@ function image_field_instance_settings_form($field, $instance) {
     '#description' => t('The alt attribute may be used by search engines, screen readers, and when the image cannot be loaded.'),
     '#weight' => 10,
   );
+  $form['alt_field_required'] = array(
+    '#type' => 'checkbox',
+    '#title' => t('<em>Alt</em> field required'),
+    '#default_value' => !isset($settings['alt_field_required']) ? 0 : $settings['alt_field_required'],
+    '#description' => t('Requiring alternative text is strongly encouraged to improve accessibility.'),
+    '#indentation' => 1,
```

Comment:
`#indentation`! :tada:

Reply:
Unfortunately this is currently broken, so I have commented it out. We have a follow-up issue to fix things in https://github.com/backdrop/backdrop-issues/issues/6090.